src/sveltekit/sdk-setup.ts

```diff
@@ -0,0 +1,433 @@
+    } else if (declaration.type === 'VariableDeclaration') {
+      const declarations = declaration.declarations;
+      declarations.forEach((declaration) => {
+        // @ts-ignore - id should always have a name in this case
+        if (!declaration.id || declaration.id.name !== 'handleError') {
+          return;
+        }
+        foundHandleError = true;
+        const userCode = declaration.init;
+        const stringifiedUserCode = userCode ? generateCode(userCode).code : '';
+        // @ts-ignore - we can just place a string here, magicast will convert it to a node
+        declaration.init = `Sentry.handleErrorWithSentry(${stringifiedUserCode})`;
+      });
+    }
+  });
+
+  if (!foundHandleError) {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    mod.exports.handleError = builders.functionCall(
+      'Sentry.handleErrorWithSentry',
+    );
+  }
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function wrapHandle(mod: ProxifiedModule<any>): void {
+  const modAst = mod.exports.$ast as Program;
+  const namedExports = modAst.body.filter(
+    (node) => node.type === 'ExportNamedDeclaration',
+  ) as ExportNamedDeclaration[];
+
+  let foundHandle = false;
+
+  namedExports.forEach((modExport) => {
+    const declaration = modExport.declaration;
+    if (!declaration) {
+      return;
+    }
+    if (declaration.type === 'FunctionDeclaration') {
+      if (!declaration.id || declaration.id.name !== 'handle') {
+        return;
+      }
+      foundHandle = true;
+      const userCode = generateCode(declaration).code;
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      mod.exports.handle = builders.raw(
+        `sequence(Sentry.sentryHandle, ${userCode.replace(
+          'handle',
+          '_handle',
+        )})`,
+      );
+      // because of an issue with magicast, we need to remove the original export
+      modAst.body = modAst.body.filter((node) => node !== modExport);
+    } else if (declaration.type === 'VariableDeclaration') {
+      const declarations = declaration.declarations;
+      declarations.forEach((declaration) => {
+        // @ts-ignore - id should always have a name in this case
+        if (!declaration.id || declaration.id.name !== 'handle') {
+          return;
+        }
+        const userCode = declaration.init;
+        const stringifiedUserCode = userCode ? generateCode(userCode).code : '';
+        // @ts-ignore - we can just place a string here, magicast will convert it to a node
+        declaration.init = `sequence(Sentry.sentryHandle, ${stringifiedUserCode})`;
+        foundHandle = true;
+      });
+    }
+  });
+
+  if (!foundHandle) {
+    // can't use builders.functionCall here because it doesn't yet
+    // support member expressions (Sentry.sentryHandle) in args
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    mod.exports.handle = builders.raw('sequence(Sentry.sentryHandle)');
```

Comment:
For future-proofness would it make sense to already make sentryHandle a factory function right now, just so that we can add config in the future.

Reply:
Opened https://github.com/getsentry/sentry-javascript/issues/7951 to track this. I agree, good point!